mempool/v1/mempool.go

```diff
@@ -483,10 +483,11 @@ func (txmp *TxMempool) addNewTransaction(wtx *WrappedTx, checkTxRes *abci.Respon
 
 		txmp.metrics.FailedTxs.With(mempool.TypeLabel, mempool.FailedAdding).Add(1)
 		reason := fmt.Sprintf(
-			"code: %d codespace: %s logs: %s postCheck error: %v",
+			"code: %d codespace: %s logs: %s local: %v postCheck error: %v",
 			checkTxRes.Code,
 			checkTxRes.Codespace,
 			checkTxRes.Log,
+			wtx.HasPeer(0), // this checks if the peer id is local
```

Comment:
I'm not 100% sure this will work as intended, but if there is no peer id, then it looks like the value will be zero. I did not check tho. cc @cmwaters

Reply:
This looks logically sound to me. It's a new transaction so there can't be any other peers but the sender

Reply:
I wonder if we should also be tracing this for recheckTx